docker/vue-app/src/components/dialog_content/IndicatorEditingDialog.js

```diff
@@ -0,0 +1,663 @@
+                            url: `${this.APIroot}formEditor/${this.currIndicatorID}/description`,
+                            data: {
+                                description: this.description,
+                                CSRFToken: this.CSRFToken
+                            },
+                            error: err => console.log('ind desciption post err', err)
+                        })
+                    );
+                }
+                if(fullFormatChanged) {
+                    indicatorEditingUpdates.push(
+                        $.ajax({
+                            type: 'POST',
+                            url: `${this.APIroot}formEditor/${this.currIndicatorID}/format`,
+                            data: {
+                                format: this.fullFormatForPost,
+                                CSRFToken: this.CSRFToken
+                            },
+                            error: err => console.log('ind format post err', err)
+                        })
+                    );
+                }
+                if(defaultChanged) {
+                    indicatorEditingUpdates.push(
+                        $.ajax({
+                            type: 'POST',
+                            url: `${this.APIroot}formEditor/${this.currIndicatorID}/default`,
+                            data: {
+                                default: this.defaultValue,
+                                CSRFToken: this.CSRFToken
+                            },
+                            error: err => console.log('ind default value post err', err)
+                        })
+                    );
+                }
+                if(requiredChanged) {
+                    indicatorEditingUpdates.push(
+                        $.ajax({
+                            type: 'POST',
+                            url: `${this.APIroot}formEditor/${this.currIndicatorID}/required`,
+                            data: {
+                                required: this.required ? 1 : 0,
+                                CSRFToken: this.CSRFToken
+                            },
+                            error: err => console.log('ind required post err', err)
+                        })
+                    );
+                }
+                if(sensitiveChanged) {
+                    indicatorEditingUpdates.push(
+                        $.ajax({
+                            type: 'POST',
+                            url: `${this.APIroot}formEditor/${this.currIndicatorID}/sensitive`,
+                            data: {
+                                is_sensitive: this.is_sensitive ? 1 : 0,
+                                CSRFToken: this.CSRFToken
+                            },
+                            error: err =>  console.log('ind is_sensitive post err', err)
+                        })
+                    );
+                }
+                if (sensitiveChanged && +this.is_sensitive === 1) {

```

Comment:
What is the "+" in front of "this" for.  I don't remember that.

Reply:
Updating this here because it's overkill (it might have been left over here from an earlier version).  When used on bools it converts them to 1/0.  The db values for these settings are 1/0 and the app tracks the checkbox selection (t/f) , so it's used in a couple places to compare them